languageserver/integration/entrypoint.go

```diff
@@ -107,5 +126,13 @@ func (i *FlowIntegration) updateEntryPointInfoIfNeeded(
 		parameters:            parameters,
 		pragmaArgumentStrings: pragmaArgumentStrings,
 		pragmaArguments:       pragmaArguments,
+		pragmaSignersStrings:  pragmaSigners,
 	}
 }
+
+func parseSignersList(signerList string) []string {
+	var re = regexp.MustCompile(`[a-zA-Z]+`)
```

Comment:
Please move this out of the function, so the regular expression is only compiled once and then re-used